Sources/BottomSheet/BottomSheet+ViewModifiers/BottomSheet+AccountForKeyboardHeight.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BottomSheet+AccountForKeyboardHeight.swift
+//  
+//
+//  Created by Robin Pel on 06/09/2022.
+//
+
+import Foundation
+
+public extension BottomSheet {
+    
+    /// Adds padding to the bottom of the main content when the keyboard appears so all of the main content is visible.
+    ///
+    /// - Note: If the height of the sheet is smaller than the height of the keyboard, this modifier will not make the content visible.
+    ///
+    /// - Parameters:
+    ///   - bool: A boolean whether the option is enabled.
+    ///
+    /// - Returns: A BottomSheet with its main content shifted up to account for the keyboard when it has appeared.
+    func accountForKeyboardHeight(_ bool: Bool = true) -> BottomSheet {
+        self.configuration.accountForKeyboardHeight = bool
+        return self
+    }
+}
```

Comment:
Please also remove the modifier on Mac (see AppleScrollBehaviour)

Reply:
But the AppleScrollBehaviour is still usable to Mac right? There is no `#if !of(macOS)` [here](https://github.com/lucaszischka/BottomSheet/blob/main/Sources/BottomSheet/BottomSheet%2BViewModifiers/BottomSheet%2BAppleScrollBehavior.swift).

Note that I have added extra not mac OS checks [here](https://github.com/RobinPelNedap/BottomSheet/blob/ipad-bottom-sheet/Sources/BottomSheet/BottomSheetView/BottomSheetView%2BHelperViews.swift#L390).

Reply:
Oh I think I got something messed up here. Yes AppleScrollBehavior on Mac and iPad is still available, because it puts the content into a ScrollView. So it still has a purpose on these platforms. However, the account for keyboard height modifier has no purpose on Mac, so I think it should be completely hidden there. I will merge this and fix it myself.